<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>400</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 189, 187)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>25</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPasswd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_signup">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>240</width>
       <height>35</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 114, 119);
color: rgb(255, 255, 255);
border : 0px;</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="userId">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Id/email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPasswdC">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>confirm password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPhone">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>phone</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/image/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_camera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 114, 119);</string>
    </property>
    <property name="text">
     <string>Please register your face...</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="facelbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_signup</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>signUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_camera</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>signUp()</slot>
  <slot>startCamera()</slot>
 </slots>
</ui>
